src/app/src/components/Navigation.tsx

```diff
@@ -129,7 +129,7 @@ export default function Navigation({
           <NavSection>
             <Logo />
             <Dropdown />
-            <NavLink href="/eval" label="Evals" />
+            <NavLink href="/evals" label="Evals" />
```

Comment:
Should think through the default behavior when we open promptfoo view and whether or want we want /eval to remain in the top nav. This may confuse people

Reply:
I think it makes sense that `promptfoo view` opens `http://localhost:15500/eval` which loads/renders the most recent eval.

Are you thinking that clicking "Evals" in the top nav and then clicking the first eval in the data grid in order to access the most recent eval isn't intuitive? The current design – where the user sees a detailed view of what the eval entails and how it performed – is definitely more information rich than the data grid.